sops/3scale-scaling.md

```diff
@@ -0,0 +1,65 @@
+# 3scale scaling on Openshift 3.11, Integreatly 1.4.x
+This guide states the known limits of 3scale capacity on Openshift along with scaling options for 3scale Components.
+
+**Note :** Pod anti-affinity rule is in place across a number of 3scale components. This may result in limited scaling options.
```

Comment:
Would be useful to provide an explanation on what anti-affinity is (maybe a link to some external docs?) and what the implications of it are i.e. limited to the number of compute nodes available